upcloud/request/managed_database.go

```diff
@@ -0,0 +1,549 @@
+package request
+
+import (
+	"encoding/json"
+	"fmt"
+	"net/url"
+	"strconv"
+	"time"
+
+	"github.com/UpCloudLtd/upcloud-go-api/upcloud"
+)
+
+/* Service Management */
+
+// CancelManagedDatabaseConnection represents a request to cancel the current query of a connection or terminate
+// the entire connection.
+type CancelManagedDatabaseConnection struct {
+	// UUID selects a managed database instance to manage queries of
+	UUID string
+	// Pid selects a connection pid to cancel or terminate
+	Pid int
+	// Terminate selects whether the connection will be forcefully terminated
+	Terminate bool
+}
+
+// RequestURL implements the Request interface
```

Comment:
Could document the package where `Request` is, as well?

Reply:
implemented